dev/tasks/exec/utils/gitignore-filter.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs' );
+const filter = require( 'gulp-filter' );
+const gitignore = require( 'parse-gitignore' );
```

Comment:
That variable name is confusing. It's a function that does something so it should contain a verb – `parseGitignore` for instance.